packages/api-client/src/components/Server/types.ts

```diff
@@ -8,4 +8,6 @@ export type ServerVariableValues = {
   [variable: string]: string
 }
 
-export type ServerVariable = OpenAPIV3_1.ServerVariableObject
+export type ServerVariable =
+  | OpenAPIV3.ServerVariableObject
+  | OpenAPIV3_1.ServerVariableObject
```

Comment:
cc @amritk finally required to maintain 3

Reply:
just temporary though